cli/src/types/version-config.ts

```diff
@@ -0,0 +1,4 @@
+export interface VersionConfig {
+  branchVersionPrefix?: string;
+  identifier?: string;
```

Comment:
Should be named `tag` to match the UI

<img width="313" alt="image" src="https://github.com/mirego/accent/assets/464900/6246050c-f430-4a02-bf4c-547856d350fa">